data/json/items/gun/762x25.json

```diff
@@ -26,7 +26,7 @@
     ],
     "valid_mod_locations": [[ "accessories", 3 ],[ "barrel", 1 ],[ "grip", 1 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]],
     "magazines": [
-      [ "762x25", [ "ppshmag", "ppshdrum" ] ]
```

Comment:
You can't rename these items now they've been pushed to `master` as this will break existing saves